src/test/java/io/jenkins/plugins/grading/AutoGraderITest.java

```diff
@@ -69,6 +74,131 @@ public void shouldCountCheckStyleWarnings() {
         assertThat(score).hasAchieved(40);
     }
 
+    /**
+     * Verifies that SpotBugs results are correctly graded.
+     */
+    @Test
+    public void shouldGradeSpotBugs(){
```

Comment:
Aus Tester Sicht ist das ein identischer Test zu dem Checkstyle Test. Sinnvoller wäre es, 2 Analysen in einem Job zu testen.